modules/oe_content_person/oe_content_person.install

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/**
+ * @file
+ * The OpenEuropa Person Content installation file.
+ */
+
+declare(strict_types = 1);
+
+/**
+ * Implements hook_install().
+ */
+function oe_content_person_install() {
+  // Configure the RDF SKOS graphs.
+  if (\Drupal::service('config.installer')->isSyncing()) {
+    return;
+  }
+
+  $graphs = [
+    'human-sex' => 'http://publications.europa.eu/resource/dataset/human-sex',
+  ];
+  \Drupal::service('rdf_skos.skos_graph_configurator')->addGraphs($graphs);
```

Comment:
Might be we should enable it globally to make it reusable later in src/PublicationsOfficeSkosGraphSetup.php and provide  oe_content_post_update_N().

Reply:
I've thought about it myself, I checked the wiki, and since this is the only content type that has gender field it makes sense to have it here